eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -456,7 +451,7 @@ class EclairRpcClient(val instance: EclairInstance)(
   case class RpcError(code: Int, message: String)
   implicit val rpcErrorReads: Reads[RpcError] = Json.reads[RpcError]
 
-  private def parseResult[T](result: JsResult[T], json: JsValue): T = {
+  private def parseResult[T: TypeTag](result: JsResult[T], json: JsValue): T = {
```

Comment:
Why is a TypeTag necessary here? Is it just for the log message? (if so why not a ClassTag?)

Reply:
Oh shoot that wasn't supposed to get in the commit...